app/modules/network_shares/locales/en.json

```diff
@@ -0,0 +1,10 @@
+{
+    "clienttab": "Network Shares",
+    "noshares": "No Network Shares",
+    "reporttitle": "Network Shares Report",
+    "name": "Name",
+    "mntfromname": "Path",
+    "fstypename": "Type",
+    "fsmtnonname": "Mount Directory",
+    "automounted": "Automount"
+}
```

Comment:
path, type and name are all available in the main locale file, so I would not duplicate them here